src/hamburger.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const hamburgerBtn = document.querySelector('.nav__hamburger-button');
+  const menu = document.querySelector('.nav__menu');
+  const menuLinks = document.querySelectorAll('.nav__link');
+
+  const toggle = () => {
+    menu.classList.toggle('nav__menu-display');
+    hamburgerBtn.classList.toggle('nav__hamburger-clicked');
+  };
+
+  menuLinks.forEach((element) => {
```

Comment:
Sounds totally like a good place for [an Event Delegation](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#event_delegation). If you use Event Delegation here you can register only one listener. Event Listeners are one of the most high-priced operations we need.